packages/pigeon/lib/dart_generator.dart

```diff
@@ -12,6 +12,8 @@ import 'generator_tools.dart';
 /// Documentation comment open symbol.
 const String _docCommentPrefix = '///';
 
+const String _varNamePrefix = '__pigeon_';
```

Comment:
Let's put a comment here explaining what this is for and why it's so non-standard.